### PR TITLE
Updated user guide, aboutus, readme and a new image for helpMessage

### LINKED ISSUE #28: Update UG with my feature

### PR DESCRIPTION
**Pull request by pzl111**
Fixes #28

### CODE REVIEW

File: docs/UserGuide.md
Comment (richiehx) [inline docs/UserGuide.md:1]: The list numbers don't need to be numbered as GFMD will auto number them, just fyi

File: docs/UserGuide.md [hunk 1]
```diff
@@ -67,8 +65,8 @@ This project is based on the [AddressBook-Level3 project](https://se-education.o
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
-</box>
+* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when co
```
Comment (richiehx) [inline docs/UserGuide.md]: For this should it be copied over to the application. instead of just co?

**Review by richiehx — COMMENTED** [review]
LGTM, just one minor issue in the User Guide.

File: docs/UserGuide.md
Comment (pzl111) [reply docs/UserGuide.md:1]: oh okay

File: docs/UserGuide.md (hunk 1 above)
Comment (pzl111) [reply docs/UserGuide.md]: whoops ok i fixed it

**Review by richiehx — APPROVED** [review]
LGTM!

**Pull request merged**